src/interactions/muller_brown.jl

```diff
@@ -0,0 +1,85 @@
+export MullerBrown
+
+@doc raw"""
+    MullerBrown(A, a, b, c, x₀, y₀)
+
+The Muller-Brown potential energy surface, witth 3 minima and 2 saddle points is given by:
+```math
+V(x,y) = \sum_{n=1}^{4} A_kexp[a_k(x-x_k^0)^2 + b_k(x-x_k^0)(y-y_k^0) + c_k(y-y_k^0)^2]
+```
+where A, a, b, c, $x_0$, $y_0$ are 4 element arrays
+"""
+#To-Do
+# A should have energy_units, in general are units working
+# Test coverage
+# Example for documentation
+# Should potential return just a number?
+
+# General interaction:
+# https://juliamolsim.github.io/Molly.jl/dev/docs/#General-interactions
+struct MullerBrown{D,T,L,F,E}
+    A::D # Array w/ units of energy
+    a::T # Array w/ units of 1/L^2
+    b::T # Array w/ units of 1/L^2
+    c::T # Array w/ units of 1/L^2
+    x₀::L # Array w/ units of L
+    y₀::L # Array w/ units of L
+    force_units::F
+    energy_units::E
+end
+
+
+function MullerBrown(A, a, b, c, x₀, y₀; 
```

Comment:
The standard values of these parameters can be given here as default arguments, i.e. `; A=SVector(-200.0, -100.0, -170.0, 15.0)u"kJ * mol^-1"`. Having them as `SVector`s will improve performance.